modules/draco/src/lib/utils/schema-attribute-utils.ts

```diff
@@ -0,0 +1,62 @@
+import {Schema, Field, FixedSizeList, getArrowTypeFromTypedArray} from '@loaders.gl/schema';
+
+import {MeshAttribute, DracoAttribute, DracoLoaderData, DracoMetadataEntry} from '../../types';
+
+export function makeSchemaFromAttributes(
+  attributes: {[attributeName: string]: MeshAttribute},
+  loaderData: DracoLoaderData,
```

Comment:
Would it be most logical to make `loaderData` the first param? All the top level schema data comes from it?